plugins/kafka/src/constants.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const KAFKA_CONSUMER_GROUP_ANNOTATION =
```

Comment:
I'm pondering if this should be plural or not. After thinking a bit, it is relatively common for keys to support named singular that support both single or multiple values. Will having it plural be confusing if it's never extended to support multiple groups?

Reply:
I'm actually working on extending it right now so I wouldn't be worried about this ever being extended to support multiple groups 😄